client.go

```diff
@@ -244,6 +258,24 @@ func IgnoreServers(s []net.IP) func(*Client) error {
 	}
 }
 
+// IgnoreServers is a function option that allow you to pass an array s of IP's
+// which the client will permit when the DHCP Server respond with a
+// dhcp4.OptionServerIdentifier (Code 54) in this IP array.
+func PermitServers(s []net.IP) func(*Client) error {
+	return func(c *Client) error {
+		c.permitServers = s
+		return nil
```

Comment:
Either:
Set the mode also. 
In IgnoreServers 2.
Or:
Create a generic set server list function. 
Change Ignore Servers to populate the list using the new method and set the mode also.